projects/aq2/sections/Kak1Hateno.celer

```diff
@@ -49,30 +49,34 @@ Kak1Hateno:
     coord: [-222.25,-435.75]
 - .dir(S) to wall:
     coord: [-210,-262]
-- BLSS to fairy:
+- BLSS .dir(SE) to Kaya Wan fairy
+- _Material<Fairy>:
     coord: [949.19, 138.58, 502.84]
 - .dir(<N)
 - _Npc<Riverbed Reward>:
     notes: Keep the weapon for IST
     coord: [823,146]
-- _Shrine::KayaWan
-- BLSS
+- _Shrine::KayaWan:
+    comment: Get core
+- BLSS .dir(E>)
 - _Shrine::DakaTuss
+- .!!(EAT BEFORE MIDAIR)
 - .dir(S) + .dir(E>>) + .dir(S)
 - _Npc::SheikaFemale<Impa>:
     coord: [1775,987]
-- Activate Shrine:
+- Activate Taloh Naeg Shrine:
     coord: [1833.5,906.1630199330921]
 - .dir(<<S) Super to camp
-- RGC slam kill 3 bokos:
-    notes: drop down and X->Y. Need the big AOE slam so keep high drop speed
```

Comment:
Revert

Reply:
Do you really need the note? I left "RGC slam kill 3 bokos". It just looks removed because there's no colon.